static/styles/portico/landing_page.css

```diff
@@ -3949,3 +3958,318 @@ nav {
     font-size: 13px;
     padding-left: 10px;
 }
+
+.feature-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 50px 4vw 10px;
+
+    .feature-half {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    .md-hide {
+        display: none;
+    }
+
+    .md-display {
+        display: block;
+    }
+
+    @media (width <= 768px) {
+        .feature-text {
+            margin-bottom: 10px;
+        }
+
+        .twitter-tweet {
+            margin: 30px auto !important;
+        }
+    }
+
+    @media (width >= 768px) {
+        display: grid;
+        grid-column-gap: 5%;
+        grid-template-columns: 43% auto;
+        max-width: 1300px;
+        align-items: center;
+        justify-items: center;
+        margin: auto;
+
+        .alternate-grid& {
+            grid-template-columns: auto 43%;
+        }
+
+        .feature-half {
+            display: block;
+        }
+
+        .md-display {
+            display: none !important;
+        }
+    }
+
+    .feature-text {
+        h1 {
+            font-size: 30px;
+            font-weight: 600;
+        }
+
+        ul {
+            list-style: none;
+            margin-left: 0;
+        }
+
+        ul li {
+            margin-bottom: 15px;
+
+            &::before {
+                float: left;
+                background-repeat: no-repeat;
+                content: "";
+                width: 18px;
+                height: 18px;
+                margin-right: 5px;
+                margin-top: 2px;
+                background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16px" height="16px" viewBox="0 0 24 24"><path d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm-1.25 17.292l-4.5-4.364 1.857-1.858 2.643 2.506 5.643-5.784 1.857 1.857-7.5 7.643z"/></svg>');
```

Comment:
Is this for the checkmark icon?  I think we can use our icon font by moving this to `static/assets/icons/`, but totally fine to do that as a follow-up.